examples/multimodal/utils.py

```diff
@@ -72,6 +72,13 @@ def add_common_args(parser):
         action='store_true',
         help='Enables chunked context (only available with cpp session).',
     )
+    parser.add_argument(
+        '--ptable_offloading',
```

Comment:
nitpick: bettern name it --mm_embedding_offloading to because ptable is underlying implementation that multimodal users may not need to know
also ptable is "prompt tuning table" instead of "position table"